recipe/meta.yaml

```diff
@@ -37,7 +37,7 @@ about:
   license: BSD-3-Clause
```

Comment:
I can't comment on the actual line, but it would be a good idea to run the actual test suite here now that we have one (probably without the visual regression tests, to avoid any deltas between our CI and the CF hardware, and particularly because of the other esoteric platforms CF supports that we don't have screenshots for and for which Playwright might not support, like x390x and PPCLE).